t/04-scheduler.t

```diff
@@ -19,6 +19,8 @@
 use strict;
 use warnings;
 
+require "OpenQA/Test/PatchDeparse.patch";
```

Comment:
when we require it in every test but only when called in a special mode – I assume using "cover" then how about including the patch on the fly in the cover call? Otherwise I am afraid someone could add a new test file soon and forget about this line and we are back with warnings

Reply:
Ok, I added this to `PERL5OPT`